addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -525,6 +553,23 @@ var MassMailingFieldHtml = FieldHtml.extend({
             }, 0);
         });
 
+        // Remove the mailing from the favorites list
+        $themeSelectorNew.on('click', '.o_mail_template_preview i.o_mail_template_remove_favorite', async (event) => {
```

Comment:
`event` is a global variable in Chrome so it's best never to use it as a variable name so as to avoid confusing errors where you forgot to include it and it uses the global `event` variable.